ml-conversational-analytic-tool/run.py

```diff
@@ -51,6 +51,10 @@ def run(annotated_filename, dataset_filename, outcome, encoding_type, model_type
     parser.add_argument('-pad', action='store_true', default=False, help='Pad total length of each pull')
 
     args = parser.parse_args()
+
+    if args.model != 'CNN' and args.model != 'LSTM':
+        raise Exception("Model must be either CNN or LSTM")
```

Comment:
I would suggest to add supported model in the argument's help string also, something like that:
`parser.add_argument('model', help='Model type to use for training, supported CNN and LSTM')`

Reply:
Done! Thanks